jgiven-timing/build.gradle

```diff
@@ -1,5 +1,7 @@
 plugins {
     id 'java-library'
+    id('jgiven-checkstyle')
+    id('jgiven-java')
```

Comment:
Stupid question: those two could not be moved into the conventions for technical reasons? Or are there some modules which would not take kindly to them?